docs/Options.md

```diff
@@ -290,6 +290,18 @@ Rust section is shown only in directories that contain `Cargo.toml` or any other
 | `SPACEFISH_RUST_COLOR` | `red` | Color of Rust section |
 | `SPACEFISH_RUST_VERBOSE_VERSION` | `false` | Show what branch is being used, if any. (Beta, Nightly) |
 
+### .NET (`dotnet`)
+
+.NET section is shown only in directories that contains a `project.json` or `global.json` file, or a file with one of these extensions: `.csproj`, `.xproj` or `.sln`.
+
+| Variable | Default | Meaning |
+| :------- | :-----: | ------- |
+| `SPACESHIP_DOTNET_SHOW` | `true` | Current .NET section |
```

Comment:
These variables should be switched to their "SPACEFISH" equivalent. 🚀🐟

Reply:
Ohhh good catch. Fixed